File: namsor/csharp/src/OSEG.NamsorExamples/PhoneCodeBatchExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class PhoneCodeBatchExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["api_key"] = "YOUR_API_KEY"};

        var personalNamesWithPhoneNumbers1 = new FirstLastNamePhoneNumberIn(
            id: "e630dda5-13b3-42c5-8f1d-648aa8a21c42",
            firstName: "Jamini",
            lastName: "Roy",
            phoneNumber: "09804201420"
        );

        var personalNamesWithPhoneNumbers = new List<FirstLastNamePhoneNumberIn>
        {
            personalNamesWithPhoneNumbers1,
        };

        var batchFirstLastNamePhoneNumberIn = new BatchFirstLastNamePhoneNumberIn(
            personalNamesWithPhoneNumbers: personalNamesWithPhoneNumbers
        );

        try
        {
            var response = new SocialApi(config).PhoneCodeBatch(
                batchFirstLastNamePhoneNumberIn: batchFirstLastNamePhoneNumberIn
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling SocialApi#PhoneCodeBatch: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: namsor/typescript-axios/src/PhoneCodeBatchExample.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const personalNamesWithPhoneNumbers1: api.FirstLastNamePhoneNumberIn = {
  id: "e630dda5-13b3-42c5-8f1d-648aa8a21c42",
  firstName: "Jamini",
  lastName: "Roy",
  phoneNumber: "09804201420",
};

const personalNamesWithPhoneNumbers = [
  personalNamesWithPhoneNumbers1,
];

const batchFirstLastNamePhoneNumberIn: api.BatchFirstLastNamePhoneNumberIn = {
  personalNamesWithPhoneNumbers: personalNamesWithPhoneNumbers,
};

new api.SocialApi(configuration).phoneCodeBatch(
  batchFirstLastNamePhoneNumberIn,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling SocialApi#phoneCodeBatch:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
